### ISSUE TITLE
Inconsistent whitespace in custom build

### ISSUE THREAD

**Issue by iver56** [driftyco/ng-cordova#744]
As you can see in the screenshot below, line breaks and indentation (tabs/spaces) is inconsistent in custom builds that I get at http://ngcordova.com/build/

![image](https://cloud.githubusercontent.com/assets/1470603/7108502/a1eaaa14-e188-11e4-82b7-bba8a9998ea2.png)

This can cause incorrect diffs on github, amongst other things

**Comment by gortok** [comment]
That's annoying. I'll add a note to normalize and make all of this consistent for a future release.  (if you'd like to speed that process along, feel free to pull and create a pull request that does this!).

**Comment by iver56** [comment]
Thanks! Is the code for the generator (which is run on the backend and seems to be hosted at http://ngcordova-builder.appspot.com/) open source? If yes, where is it?

**Comment by gortok** [comment]
You're right, I read this issue differently.  I don't know where the code for the custom builder is, but I'll find it and let you know. I ways assumed it was part of the build tasks for this repo.
